database/src/main/kotlin/no/nav/su/se/bakover/database/søknadsbehandling/SøknadsbehandlingPostgresRepo.kt

```diff
@@ -71,6 +64,12 @@ internal class SøknadsbehandlingPostgresRepo(
                 is Søknadsbehandling.Underkjent -> lagre(søknadsbehandling, tx)
                 is Søknadsbehandling.Iverksatt -> lagre(søknadsbehandling, tx)
                 is LukketSøknadsbehandling -> lagre(søknadsbehandling, tx)
+            }.also {
+                grunnlagsdataOgVilkårsvurderingerPostgresRepo.lagre(
```

Comment:
Tilsvarende kommentar her som for revurdering mtp utakt mellom domenemodellen vs repo i forhold til gyldige operasjoner og tilstander.